site2/website/release-notes.md

```diff
@@ -1,6 +1,300 @@
 
 ## Apache Pulsar Release Notes
 
+### 2.10.0
+#### 2022-02-28
+
+### Important notice
+- Remove -XX:-ResizePLAB JVM option which degrades performance on JDK11 [#12940](https://github.com/apache/pulsar/pull/12940)
+- Enable TCP keepAlive flag on the sockets [#12982](https://github.com/apache/pulsar/pull/12982)
+- Reduce the time it takes for namespace bundle unloading to time out [#12995](https://github.com/apache/pulsar/pull/12995)
+- Align configurations defaults between default file and Java object (broker.conf, proxy.conf, websocket.conf) [#13272](https://github.com/apache/pulsar/pull/13272)
+- [PIP 118] Do not restart brokers when ZooKeeper session expires as default [#13341](https://github.com/apache/pulsar/pull/13341)
+- [PIP 119] Enabled consistent hashing by default on KeyShared subscriptions dispatcher [#13352](https://github.com/apache/pulsar/pull/13352)
+- [PIP 120] Enabled client memory limit controller by default [#13344](https://github.com/apache/pulsar/pull/13344)
+- Make Docker images non-root, by default, and OpenShift compliant [#13376](https://github.com/apache/pulsar/pull/13376)
+- [PIP 122] Change loadBalancer default loadSheddingStrategy to ThresholdShedder [#13733](https://github.com/apache/pulsar/pull/13733)
+- Fix netcat returning early for probe [#14088](https://github.com/apache/pulsar/pull/14088)
+
+### PIPs
+- [PIP 79] Added lazy-loading feature to PartitionedProducer [#10279](https://github.com/apache/pulsar/pull/10279)
+- [PIP 84] Pulsar client: Redeliver command add epoch [#10478](https://github.com/apache/pulsar/pull/10478)
+- [PIP 86] Pulsar Functions: Preload and release external resources [#13205](https://github.com/apache/pulsar/pull/13205)
+- [PIP 97] Update Authentication Interfaces to Include Async Authentication Methods [#12104](https://github.com/apache/pulsar/pull/12104)
```

Comment:
This PIP is not completed for 2.10. I reverted the main changes https://github.com/apache/pulsar/pull/14424. We can remove the line here.

```suggestion
```